src/main/java/org/lwjglx/openal/AL.java

```diff
@@ -64,6 +64,10 @@ public static void create(String deviceArguments, int contextFrequency, int cont
 
         long deviceHandle = org.lwjgl.openal.ALC10.alcOpenDevice(defaultDevice);
 
+        if (deviceHandle == 0) {
+            throw new org.lwjgl.LWJGLException("Could not open ALC device");
```

Comment:
The mappings should only throw the lwjglx exceptions, not lwjgl - change this here and remove the throws clauses above. The class references are remapped from `x` to no `x` in an asm transformer later

Reply:
Changed to `lwgjlx` variant. I think we still need the throws above because the create calls are chained.